Source/Core/Core/HW/Memmap.cpp

```diff
@@ -333,6 +339,9 @@ bool InitFastmemArena()
 
     if (base != view)
     {
+      PanicAlertFmt("Memory::InitFastmemArena(): Failed to map memory region at 0x{:08X} (size "
```

Comment:
Nitpicking, but can the `(size ` be moved to the next line to make this more pleasant? (I assume clang-format is what reformatted it like this.)

Reply:
Yup that was clang-format. Should be nicer now.